ci-operator/config/openshift/microshift/openshift-microshift-main.yaml

```diff
@@ -3,10 +3,6 @@ base_images:
     name: "4.14"
     namespace: ocp
     tag: cli
-  libvirt-installer:
-    name: "4.14"
-    namespace: ocp
-    tag: libvirt-installer
```

Comment:
A little confused, this image was necessary to interact with the custom gcp test instance because it's preconfigured with `mock-nss`, which is what allowed us to execute gcloud ssh/scp commands from within the step.  I see it's been removed throughout - is it still a dependency for infra steps?

Reply:
From my testing it looks like it works no problem without it. Also I don't think we're using libvirt anywhere. `gcloud` is obtained in `build_root` from the same repository that `libvirt-installer` installs it from

Reply:
Looking thru the existing step scripts, we've dropped the mock-nss script usage at some point, so that explains my confusion.  That's interesting b/c (at least at first) we couldn't get ssh client ops to work without spoofing a user.  That was a workaround to OCP's randomized user, which isn't injected into /etc/passwd and causes ssh to fail.  🤷🏻 Anyway, if it passes it passes.